src/main/java/org/utplsql/api/DBHelper.java

```diff
@@ -52,4 +54,80 @@ public static String getCurrentSchema(Connection conn) throws SQLException {
         }
     }
 
+    /**
+     * Check the utPLSQL version compatibility.
+     * @param conn the connection
+     * @return true if the requested utPLSQL version is compatible with the one installed on database
+     * @throws SQLException any database error
+     */
+    public static boolean versionCompatibilityCheck(Connection conn, String requested, String current)
+            throws SQLException {
+        CallableStatement callableStatement = null;
+        try {
+            callableStatement = conn.prepareCall("BEGIN ? := ut_runner.version_compatibility_check(?, ?); END;");
+            callableStatement.registerOutParameter(1, Types.SMALLINT);
+            callableStatement.setString(2, requested);
+
+            if (current == null)
+                callableStatement.setNull(3, Types.VARCHAR);
+            else
+                callableStatement.setString(3, current);
+
+            callableStatement.executeUpdate();
+            return callableStatement.getInt(1) == 1;
+        } catch (SQLException e) {
+            if (e.getErrorCode() == 6550)
+                return false;
+            else
+                throw e;
+        } finally {
+            if (callableStatement != null)
+                callableStatement.close();
+        }
+    }
+
+    public static boolean versionCompatibilityCheck(Connection conn, String requested)
+            throws SQLException {
+        return versionCompatibilityCheck(conn, requested, null);
+    }
+
+    public static boolean versionCompatibilityCheck(Connection conn)
+            throws SQLException {
+        return versionCompatibilityCheck(conn, UTPLSQL_VERSION);
+    }
+
+    /**
+     * Enables the dbms_output buffer.
+     * @param conn the connection
+     * @param bufferLen the buffer length
+     */
+    public static void enableDBMSOutput(Connection conn, int bufferLen) {
+        try (CallableStatement call = conn.prepareCall("BEGIN dbms_output.enable(?); END;")) {
```

Comment:
I would use NULL as max value of the buffer.
As per [Oracle doc](https://docs.oracle.com/cd/E11882_01/appdev.112/e40758/d_output.htm#ARPLS67301), ENABLE(NULL) makes the buffer size unlimited.

Reply:
Great, I will remove the one with the size parameter, I think we don't need that anyways...